src/scrabble.js

```diff
@@ -1,5 +1,58 @@
-function scrabble() {
+function scrabble(scrabbleWord) {
+  if (!scrabbleWord) return 0
+  const libary = {
+    a: 1,
+    e: 1,
+    i: 1,
+    o: 1,
+    u: 1,
+    l: 1,
+    n: 1,
+    r: 1,
+    s: 1,
+    t: 1,
+    d: 2,
+    g: 2,
+    b: 3,
+    c: 3,
+    m: 3,
+    p: 3,
+    f: 4,
+    h: 4,
+    v: 4,
+    w: 4,
+    y: 4,
+    k: 5,
+    j: 8,
+    x: 8,
+    q: 10,
+    z: 10
+  }
+  let sum = 0
+  scrabbleWord = scrabbleWord.toLowerCase()
 
+  for (let i = 0; i < scrabbleWord.length; i++) {
+    sum += libary[scrabbleWord[i]] || 0
+
+    // double or triple word to come here ( look at if statements )
+  }
+  if (
```

Comment:
This could potentially be another function embedded into scrabble()?

Reply:
Seconded, you could write a separate function, passing in the score and then returning the new score with added bonus points. But very cool implementation